src/lib.rs

```diff
@@ -408,6 +445,44 @@ fn parse_wasm_features(features: &str) -> Result<wasmparser::WasmFeatures> {
         memory64: false,
     })
 }
+
+fn parse_wasi_modules(modules: &str) -> Result<WasiModules> {
+    let modules = modules.trim();
+    match modules {
+        "all" => Ok(WasiModules::default()),
+        "all-experimental" => Ok(WasiModules::all()),
```

Comment:
Hmm, I'm not sure how I feel about the drift between API and command line parameters here. I worry there might be some risk of someone learning the CLI parameters and then assuming `WasiModules::all()` means the same as `--wasi-modules=all`.

Reply:
`WasiModules::all()` is now gone; `"default"` corresponds to `WasiModules::default()` and `"-default"` corresponds to `WasiModules::none()`.